docs/keto/modeling/create-permission-model.mdx

```diff
@@ -0,0 +1,377 @@
+---
+id: create-permission-model
+title: How to create a permission model with Ory Permission Language
+sidebar_label: Create a permission model
+toc_max_heading_level: 4
+---
+
+# Create a permission model
+
+```mdx-code-block
+import Mermaid from "@site/src/theme/Mermaid"
+```
+
+This section guides you through creating your first permission model using the Ory Permission Language (OPL).
+
+## What is a permission model?
+
+A permission model is a set of rules that define which relations are checked in the database during a permission check.
+
+Permission checks are answered based on:
+
+- The data available in the database, for example: `User:Bob is owner of Document:X`
```

Comment:
`User:Bob is owner of Document:X` - It seemed like you wanted this to be highlighted with Prism but it didn't work. First of all, it works only for code blocks and not for in-line code :( Even when I tried turning this into a code block, the highlighting still didn't work. Probably the way you put the relationship wasn't recognized by prism. Here are examples Patrik added recently: 

````
```keto-relation-tuples
namespace:object#relation:subject
// comment
```

```keto-relationships
namespace:subject is relation of object
is namespace:subject allowed to permission on object?
// comment
```
````

Reply:
Strange, I don't see see the highlighting either way, not even on https://www.ory.sh/docs/keto/#relationships. @zepatrik is that highlighting broken, or how can I use it?